understanding/parsing.html

```diff
@@ -55,92 +53,75 @@
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 4.1.1:</span>Parsing (Obsolete and removed) (Level )
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 4.1.1:</span>構文解析 (廃止・削除)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>This criterion was originally adopted to address problems that assistive technology
-                        had directly parsing HTML. Assistive technology no longer has any need to directly
-                        parse HTML. Consequently, these problems either no longer exist or are addressed by
-                        other criteria. This criterion no longer has utility and is removed.
+                     <p class="note-title marker">注記</p>
+                     <p>この達成基準はもともと、支援技術が直接 HTML を構文解析する際の問題に対処するために採用された。支援技術は、もはや直接 HTML を構文解析する必要がなくなった。その結果、これらの問題はもはや存在しない、又は他の達成基準によって対処されている。この達成基準はもはや有用ではないため、削除された。
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Assistive technology can properly present page content.</dd>
+                  <dt>目標</dt>
+                  <dd>支援技術はページコンテンツを適切に表示できる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Create web pages according to specifications.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>仕様に従ってウェブページを作成する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People can browse web content more easily with their assistive technology.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>支援技術を使用すると、人々はウェブコンテンツをより簡単に閲覧できる。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <div class="wcag22"> <span class="new-version">New in WCAG 22: </span>
+               <h2>意図</h2>
+               <div class="wcag22"> <span class="new-version">WCAG 2.2 の新基準:</span>
```

Comment:
New を「新基準」と訳すと、新たな達成基準のように見えてしまう印象を受けました。意訳気味ですが、以下、いかがでしょうか？

```suggestion
               <div class="wcag22"> <span class="new-version">WCAG 2.2 での新しい変更:</span>
```